crm-app/src/constants.js

```diff
@@ -0,0 +1,9 @@
+const constants = {
+	ADD_CUSTOMER: "ADD_CUSTOMER",
```

Comment:
A single quotation would be better here.

Reply:
Fixed